cypress/support/step_definition/group/commonGroupSteps.spec.js

```diff
@@ -0,0 +1,47 @@
+const { When, Then } = require('@badeball/cypress-cucumber-preprocessor')
```

Comment:
1) Here also can you change the name of the folder to -> 'class_management' instead of 'group'
2) And the step definition file name would be ->'commonClassesSteps.spec.js'